pyne/srim_read.py

```diff
@@ -0,0 +1,165 @@
+"""This module is for SRIM files. Currently defines a
+SRIM class, reads a srim file.
+"""
+
+from warnings import warn
+from pyne.utils import QAWarning
+import numpy as np
+
+warn(__name__ + " is not yet QA compliant.", QAWarning)
+
+
+class srim(object):
+    """srim class includes srim specific variables"""
+
+    def __init__(self, ion='', ion_mass=0, target_density=0,
+                 target_comp=None, date='', bragg_corr=0,
+                 filename='',ions=None,stop_pow_ele=None,stop_pow_nuc=None,
+                 ion_units=None,proj_ranges=None,proj_range_units=None,
+                 long_straggs=None,long_stragg_units=None,lat_straggs=None,
+                 lat_stragg_units=None):
+        """Define srim specific variables """
```

Comment:
This docstring should include the meaning and type of all of the arguments and keyword arguments (except `self`, which doesn't need to be documented.)